Symfony/CS/Tests/AbstractFixerTest.php

```diff
@@ -17,7 +17,7 @@
 class AbstractFixerTest extends \PHPUnit_Framework_TestCase
 {
     /**
-     * @expectedException        LogicException
+     * @expectedException        \LogicException
```

Comment:
If you leave at the `\` at the beginning the IDE will assume the current namespace, unless there is a class in the namespace it will not able to find it thus gives a warning.